deep_qa/training/trainer.py

```diff
@@ -282,6 +291,10 @@ def train(self):
         # Then we build the model and compile it.
         logger.info("Building the model")
         self.model = self._build_model()
+        if self.num_gpus > 1:
+            with tensorflow.device("/cpu:0"):
```

Comment:
Yeah this was from when I was playing around with using a global cpu scope before I realised I could use that conditional scope thing - I am not sure if we need to wrap when we actually call `build_model`, just in case some of our layers are actually build when we call this (rather than being built in model.compile). Regardless, this is in the wrong place even if we do need to do this.